docs/submission-evaluation.md

```diff
@@ -0,0 +1,25 @@
+###### tags: `tealgarden`
+
+# Submission Proposal Evaluation
+
+Teal.Garden is our community platform for self-managed & distributed teams. When collecting content & insights, we want to avoid tool developers to use teal.garden as a simple marketing platform. Also, teal.garden is not a collection of installation guides.
+
+How do we clarify which practices & processes count as a valid submission to teal.garden?
```

Comment:
these are questions to ourselves, should be rewritten for the hackathon participants.